### PR TITLE
Simplify SVG component implementation

### PR DESCRIPTION
**Pull request by keithluchtel**

### CODE REVIEW

File: libs/component-library/src/lib/svg/svg.tsx [hunk 1]
```diff
@@ -1,57 +1,23 @@
...
-    | 'color'
-    | 'fill'
-    | 'opacity'
-    | 'fontSize'
-    | 'fontStyle'
-    | 'fontWeight'
-    | 'letterSpacing'
-  >
->([spacing]);
+const RestyledSvg = createRestyleComponent<FxSvgProps, FxTheme>([spacing], Svg);
```
Comment (buchansm) [inline libs/component-library/src/lib/svg/svg.tsx:17]: Shouldn't this include `[color, spacing]`?

Comment (JDMathew) [reply libs/component-library/src/lib/svg/svg.tsx:17]: In this case I think it's not needed as it's coming from the `useTheme` hook instead and being applied to the SVG using the fill prop. I could be wrong.

Comment (JDMathew) [reply libs/component-library/src/lib/svg/svg.tsx:17]: In this case I think it's not needed as it's coming from the useTheme hook instead and being applied to the SVG using the fill prop. I could be wrong.

**Review by JDMathew — APPROVED** [review]
LGTM

File: libs/component-library/src/lib/svg/svg.tsx (hunk 1 above)
Comment (keithluchtel) [reply libs/component-library/src/lib/svg/svg.tsx:17]: Yep, color is getting converted with theme values from `useTheme`

**Pull request merged**